src/components/signInForm/signInHook.ts

```diff
@@ -0,0 +1,21 @@
+import * as yup from 'yup';
+import { useTranslation } from 'react-i18next';
+
+export const useSignInSchema = () => {
+  const { t } = useTranslation();
+  const signInSchema = yup
+    .object()
+    .shape({
+      email: yup
+        .string()
+        .email(t('validation.email'))
+        .required('Please enter your credentials'),
+      password: yup
+        .string()
+        .min(8, 'Password format is incorrect')
```

Comment:
strings and magic numbers should be extracted

Reply:
fixed